__tests__/markdown-generator.test.ts

```diff
@@ -148,6 +154,18 @@ describe("Code", () => {
         expect(result).toMatchSnapshot();
     });
 
+    it("inline with escape", () => {

```

Comment:
We should have a consistent convention about test names (should we start capital letter or lower case).